openstack/keppel/values.yaml

```diff
@@ -268,16 +271,30 @@ pgmetrics:
         - counts:
             usage: "GAUGE"
             description: "Total number of manifests where indexing in Clair failed with a specific error message"
+    keppel_trivy_scan_error:
+      query: >
+        SELECT message, vuln_status, COUNT(*) AS counts FROM trivy_security_info GROUP BY message, vuln_status
```

Comment:
Maybe we should rename the metric to keppel_trivy_scan_messages or similar because clair returned the same information for a long time and maybe you can build nice dashboards with it to show the distribution of things.

```
postgres@127:keppel> SELECT message, vuln_status, COUNT(*) AS counts FROM trivy_security_info GROUP BY message, vuln_status;
+----------------------------------------------------------------------------------------------------------------------------------+-------------+--------+
| message                                                                                                                          | vuln_status | counts |
|----------------------------------------------------------------------------------------------------------------------------------+-------------+--------|
|                                                                                                                                  | Low         | 4      |
|                                                                                                                                  | Clean       | 298    |
| vulnerability scanning is not supported for uncompressed image layers above 10 GiB                                               | Unsupported | 16     |
|                                                                                                                                  | Unknown     | 42     |
| vulnerability scanning is not supported for blob layers with media type "application/vnd.aquasec.trivy.db.layer.v1.tar+gzip"     | Unsupported | 13     |
|                                                                                                                                  | High        | 2060   |
|                                                                                                                                  | Critical    | 4399   |
|                                                                                                                                  | Pending     | 1      |
| vulnerability scanning is not supported for blob layers with media type "application/vnd.cncf.helm.chart.content.v1.tar+gzip"    | Unsupported | 9      |
| vulnerability scanning is not supported for blob layers with media type "application/tar+gzip"                                   | Unsupported | 2      |
|                                                                                                                                  | Medium      | 177    |
| vulnerability scanning is not supported for blob layers with media type "application/vnd.aquasec.trivy.javadb.layer.v1.tar+gzip" | Unsupported | 3      |
+----------------------------------------------------------------------------------------------------------------------------------+-------------+--------+
```